### PR TITLE
OHIF-301: Add sorting to worklist series

### PR DESCRIPTION
**Pull request by igoroctaviano**
### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by JamesAPetts** [comment]
Tested and LGTM, could you resolve the conflicts?

**Comment by igoroctaviano** [comment]
> Tested and LGTM, could you resolve the conflicts?

@JamesAPetts done.

File: extensions/default/src/DicomWebDataSource/qido.js [hunk 1]
```diff
@@ -123,9 +124,9 @@ async function search(
 export function seriesInStudy(dicomWebClient, studyInstanceUID) {
   // Series Description
   // Already included?
-  const commaSeparatedFields = ['0008103E'].join(',');
```
Comment (swederik) [inline extensions/default/src/DicomWebDataSource/qido.js:126]: Do not replace '0008103E' with '00080021'. We want them both. Just make it `['0008103E', '00080021'].join(',')`

Comment (swederik) [reply extensions/default/src/DicomWebDataSource/qido.js:126]: Nevermind, fixed via Github UI

**Pull request merged**